ddtrace/writer.py

```diff
@@ -40,8 +39,13 @@ def write(self, spans=None, services=None):
         if spans:
             self._traces.add(spans)
 
-        if services:
-            self._services.add(services)
+        if services is not None:
+            deprecation(
+                name='write(services)',
+                message='Writing services has been replaced with specifying the'
```

Comment:
Same as the other deprecation/continuation comment, please use parenthesis around multi-line/continuation strings.